woocommerce.world

```diff
@@ -0,0 +1 @@
y":0,"z":3},"rotation":{"x":0,"y":0,"z":0},"scale":{"x":1,"y":1,"z":1}}},{"name":"visible","props":{"visible":true}},{"name":"wooCommerce","props":{"wooCommerceProducts":[{"title":"ThinkPad X1 Carbon Gen 9 Intel (14\")","description":"With the Intel® Evo™ platform, the ThinkPad X1 Carbon Gen 9 laptop delivers a powerhouse combination of performance, responsiveness, battery life, and stunning visuals. With up to 11th Gen Intel® Core™ i7 vPro® processors, you can count on an exceptional experience, anywhere.\n","storeUrl":"https://xrengine.000webhostapp.com/product/12","value":12,"label":"ThinkPad X1 Carbon Gen 9 Intel (14\")","media":[{"url":"https://xrengine.000webhostapp.com/wp-content/uploads/2021/11/image-3-300x200.jpg","format":"jpg","extendType":"image"},{"url":"https://xrengine.000webhostapp.com/wp-content/uploads/2021/11/razer_laptop.glb","format":"glb","extendType":"model"},{"url":"https://xrengine.000webhostapp.com/wp-content/uploads/2021/11/Sample.mp4","format":"mp4","extendType":"video"}]}],"wooCommerceDomain":"https://xrengine.000webhostapp.com","wooCommerceConsumerKey":"ck_3bf5cd65e42df77005d1bd67c62ea796ef7ce0df","wooCommerceConsumerSecret":"cs_fad09abbbde3976eac91a2f75548508709fd04a1","wooCommerceProductId":12,"wooCommerceProductItemId":1,"wooCommerceProductItems":[{"value":0,"label":"Image 1","media":{"url":"https://xrengine.000webhostapp.com/wp-content/uploads/2021/11/image-3-300x200.jpg","format":"jpg","extendType":"image"}},{"value":1,"label":"Model 1","media":{"url":"https://xrengine.000webhostapp.com/wp-content/uploads/2021/11/razer_laptop.glb","format":"glb","extendType":"model"}},{"value":2,"label":"Video 1","media":{"url":"https://xrengine.000webhostapp.com/wp-content/uploads/2021/11/Sample.mp4","format":"mp4","extendType":"video"}}],"extendType":"model","extend":{"src":"https://xrengine.000webhostapp.com/wp-content/uploads/2021/11/razer_laptop.glb","textureOverride":"","castShadow":false,"receiveShadow":false,"loop-animation":{"activeClipIndex":-1,"hasAvatarAnimations":false}}}},{"name":"interact","props":{"interactable":true,"interactionType":"infoBox","interactionText":"ThinkPad X1 Carbon Gen 9 Intel (14\")","interactionThemeIndex":0,"interactionName":"ThinkPad X1 Carbon Gen 9 Intel (14\")","interactionDescription":"With the Intel® Evo™ platform, the ThinkPad X1 Carbon Gen 9 laptop delivers a powerhouse combination of performance, responsiveness, battery life, and stunning visuals. With up to 11th Gen Intel® Core™ i7 vPro® processors, you can count on an exceptional experience, anywhere.\n","interactionImages":["https://xrengine.000webhostapp.com/wp-content/uploads/2021/11/image-3-300x200.jpg"],"interactionVideos":["https://xrengine.000webhostapp.com/wp-content/uploads/2021/11/Sample.mp4"],"interactionUrls":["https://xrengine.000webhostapp.com/product/12"],"interactionModels":["https://xrengine.000webhostapp.com/wp-content/uploads/2021/11/razer_laptop.glb"]}}],"parent":"2266BED7-6CC4-48A6-95DD-9BCD3CF9EAFC","index":7}}}
```

Comment:
This is the Woocommerce world file. Please be careful about the CORS when fetching the assets.